database/migrations/2020_05_04_150000_create_bookings_table.php

```diff
@@ -7,31 +7,31 @@
 use Illuminate\Support\Facades\Schema;
 use Tipoff\Bookings\Models\BookingCategory;
 use Tipoff\Bookings\Models\BookingStatus;
+use Tipoff\Bookings\Models\Rate;
 
 class CreateBookingsTable extends Migration
 {
     public function up()
     {
         Schema::create('bookings', function (Blueprint $table) {
             $table->id();
-            $table->string('slot_number')->nullable();
-            $table->integer('total_participants')->nullable();
+            $table->string('slot_number');
+            $table->integer('total_participants');
             $table->unsignedInteger('total_amount');
-            $table->unsignedInteger('amount')->nullable();
-            $table->unsignedInteger('total_taxes')->nullable();
-            $table->unsignedInteger('total_fees')->nullable();
-//            $table->morphs('variation')->nullable();
-//            $table->morphs('experience')->nullable();
-//            $table->morphs('order')->nullable();
-            $table->unsignedBigInteger('booking_category_id');
-            $table->foreign('booking_category_id')->references('id')->on('booking_categories');
-//            $table->unsignedBigInteger('booking_status_id');
-//            $table->foreign('booking_status_id')->references('id')->on('booking_status');
-//            $table->morphs('agent')->nullable();
-//            $table->morphs('user')->nullable();
-//            $table->morphs('subject')->nullable();
-            $table->datetime('proccessed_at')->nullable();;
-            $table->datetime('canceled_at')->nullable();;
+            $table->unsignedInteger('amount');
+            $table->unsignedInteger('total_taxes');
+            $table->unsignedInteger('total_fees');
+            $table->morphs(app('experience'));
```

Comment:
As stated before, there's gonna need to be some updates to support all the different morph types. This will pretty much break any future tests going forward for this package.

Reply:
Changing types is a part of ticket. Its requiring a couple of steps but at the end behavior should be comparable.